Exercises/ResistorColor/ResistorColor.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercises.ResistorColor
+{
+    internal class ResistorColor
+    {
+        public static int ColorCode(string color)
+        {
+            string[] listColors = Colors();
```

Comment:
Visto que o tipo já mostra que é uma lista, acho que poderíamos chamar só de " colors". Que acha?

```suggestion
            string[] colors = Colors();
```

Reply:
Eu havia deixado só colors, mas achei q no for ficava meio confuso.